docs/api.md

```diff
@@ -29,18 +29,18 @@ localForage includes a localStorage-backed fallback store for browsers with no I
 
 ```bash
 # Install via npm:
-npm install localforage
+npm i localforage
```

Comment:
Please revert this one to `install`, as it's easier to read. The shorthand is useful for typing, but since this is copy+paste it's better to optimise for reading than typing 😄